demos/signature/EVP_DSA_Signature_demo.c

```diff
@@ -268,7 +268,7 @@ static int demo_verify(OSSL_LIB_CTX *libctx,
 
 int main(void)
 {
-    int result = 0;
+    int result = EXIT_FAILURE;
```

Comment:
Why change `result` to `ret` in `EVP_PKEY_EC_keygen.c` but not here?

Reply:
Changed it everywhere.. It used rv, result and ret.